src/config.js

```diff
@@ -262,57 +263,75 @@ const siteFacebookPage = 'https://www.facebook.com/Sharetribe/';
 // You should create one to track social sharing in Facebook
 const facebookAppId = null;
 
-// Google Maps API key is needed for static map images.
-const googleMapsAPIKey = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
-
-const coordinates = {
-  // If true, obfuscate the coordinates of the listings that are shown
-  // on a map.
-  fuzzy: false,
-
-  // Default zoom level when showing a single circle on a Map. Should
-  // be small enough so the whole circle fits in.
-  fuzzyDefaultZoomLevel: 14,
-
-  // When fuzzy === true, the coordinates on the Map component are
-  // obfuscated and a circle is shown instead of a marker. To decide a
-  // proper value for the offset and the radius, see:
-  //
-  // https://gis.stackexchange.com/a/8674
-
-  // Amount of maximum offset in meters that is applied to obfuscate
-  // the original coordinates. The actual value is random, but the
-  // obfuscated coordinates are withing a circle that has the same
-  // radius as the offset.
-  coordinateOffset: 500,
+const maps = {
```

Comment:
We could call this `map` as the application is using one map at a time. But I guess `maps` makes sense too.

Reply:
Well, you sort of configure all the maps in the app. And nothing prevents using two or more maps at a time. But mainly `maps` is a bit more greppable than `map`.